SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+The Uptane community is committed to maintaining a reliable and consistent Standard. If you believe you have identified errata in the Uptane Standard, please follow these guidelines for responsible disclosure.
```

Comment:
```suggestion
The Uptane community is committed to maintaining a reliable and consistent Standard. If you believe you have identified errata—including security issues—in the Uptane Standard, please follow these guidelines for responsible disclosure.
```

Reply:
This is fine with me. @trishankatdatadog thanks for the assist. Can you merge? It's been reviewed multiple times